simple_history/models.py

```diff
@@ -307,5 +318,5 @@ def __init__(self, model):
 
     def __get__(self, instance, owner):
         values = (getattr(instance, f.attname)
-                  for f in self.model._meta.fields)
+                  for f in HistoricalRecords.get_fields(self.model))
```

Comment:
Good point nonetheless. I found another case I had missed, in HistoryManager.most_recent.
I realize I do not know how to get the HistoricalRecord object from the instance; I'm now thinking the get_fields method should actually be in the HistoryManager and not the HistoricalRecord. Would you agree?